app/views/carts/show.html.erb

```diff
@@ -14,8 +14,8 @@
   </tr>
 </thead>
 <tbody>
-  <%= render partial: "partials/cart_items", collection: @cart.items, as: :item %>
+  <%= render partial: "cart_items", collection: @cart.items, as: :item %>
 </tbody>
 </table>
 
-<h2><%= "Total Price: " %><%= number_to_currency(@cart.total_price) %></h2>
+<h2><%= "Total Price: $#{@cart.total_price}.00" %></h2>
```

Comment:
Unsure why we're not wanting to use the number to currency method here. Any specific reason?